quantus/metrics/robustness/avg_sensitivity.py

```diff
@@ -307,6 +312,15 @@ def evaluate_batch(
                 **self.perturb_func_kwargs,
             )
 
+            changed_prediction_indexes = (
```

Comment:
rename changed_prediction_indexes to changed_prediction_indices

Reply:
done